components/service/glean/src/test/java/mozilla/components/service/glean/private/TimespanMetricTypeTest.kt

```diff
@@ -115,16 +115,16 @@ class TimespanMetricTypeTest {
         )
 
         // Record a timespan.
-        metric.start(this)
-        metric.stopAndSum(this)
+        val timerId = metric.start()
+        metric.stopAndSum(timerId)
 
         // Check that data was properly recorded in the second ping.
         assertTrue(metric.testHasValue("store2"))
         assertTrue(metric.testGetValue("store2") >= 0)
     }
 
     @Test
-    fun `Records an error if started twice`() {
+    fun `Starting twice should generate different timerId`() {
```

Comment:
I thought starting twice was supposed to generate an error? (and indeed it records an error, doesn't it?)

Reply:
True, fixed this.